rdkit/Chem/Descriptors.py

```diff
@@ -181,6 +181,21 @@ def MinAbsPartialCharge(mol, force=False):
 MinAbsPartialCharge.version = "1.0.0"
 
 from rdkit.Chem.EState.EState import MaxEStateIndex, MinEStateIndex, MaxAbsEStateIndex, MinAbsEStateIndex
+from rdkit.Chem.QED import qed
+
+
+def _FingerprintDensity(mol, func, *args, **kwargs):
```

Comment:
I assume this is for the morgan-2 fingerprints. The count vector form must go to 3, as there are 3 fragments created around each atom. For `C` it is only one (1/1), for `CC` it's two (4/2 theoretically), for `CCC` it's (3+2+3/3 theoretically). In reality it's a bit more complicated, as we take the bonding to the next partner into account and, I think, the terminal atoms are treated somewhat differently; (2 + 1)/2 for CC, (2 + 1 + 1 + 2)/3 for CCC) . For 'C'*n and n sufficiently large, the counts are: (n-2, n-4, n-6, 5*2), so we get (3*n - 2) / n = 3 - 2/n.

That makes the count vector relatively uninteresting. Even more complex structures, e.g. `'CNCCNNCCNC'*10` have the same asymptotic behaviour. The counts vector based descriptor is only a measure of how many terminal groups and what percentage of the structure they make. 

The bit vector form is more interesting; it is a measure of structural diversity. Here, the asymptotic for `'C'*n` is 8/n so it will approach 0 with increasing chain length.